src/main/scala/org/apache/spark/sql/redis/RedisPersistence.scala

```diff
@@ -13,9 +13,10 @@ trait RedisPersistence[T] extends Serializable {
 
   def load(pipeline: Pipeline, key: String, requiredColumns: Seq[String]): Unit
 
-  def encodeRow(value: Row): T
+  def encodeRow(key: (String, String), value: Row): T
```

Comment:
Could you please add a javadoc for these methods. It is now not obvious why a key is a tuple.

Reply:
sure